simphony/core/keywords.py

```diff
@@ -36,7 +36,7 @@
         name='ElectrostaticField',
         definition='Electrostatic field',  # noqa
         key='ELECTROSTATIC_FIELD',
-        shape=[],
+        shape=[1],
```

Comment:
@tuopuu see this PR please: https://github.com/simphony/simphony-metadata/pull/48